proto/src/determined/trial/v1/trial.proto

```diff
@@ -320,3 +321,68 @@ message TrialSourceInfo {
   // Type for this trial_source_info
   TrialSourceInfoType trial_source_info_type = 5;
 }
+
+// Flat run respresentation.
+message Run {
+  // The id of the run.
+  int32 id = 1;
+  // The time the run was started.
+  google.protobuf.Timestamp start_time = 2;
+  // The time the run ended.
+  google.protobuf.Timestamp end_time = 3;
+  // The current state of the run(trial).
+  State state = 4;
+  // The tags of the run.
+  string tags = 5;
+  // The total size of checkpoints.
+  int64 checkpoint_size = 6;
+  // The count of checkpoints.
+  int32 checkpoint_count = 7;
+  // The type of searcher for the experiment.
+  string searcher_type = 8;
+  // The searcher metric name for the experiment.
+  string searcher_metric = 9;
+  // Signed searcher metrics value.
+  double searcher_metric_value = 10;
+  // The id of external run
+  int32 external_run_id = 11;
+  // Trial hyperparameters.
+  string hyperparameters = 12;
+  // summary metrics.
+  string metrics = 13;
+  // The id of the experiment linked to the run.
+  int32 experiment_id = 14;
+  // The id of the user who created the parent project.
+  int32 owner_id = 15;
+  // Original id of a forked or continued experiment.
+  google.protobuf.Int32Value forked_from = 16;
+  // The id of external experiment
+  int32 external_experiment_id = 17;
+  // The resource pool the experiment was created in.
+  string resource_pool = 18;
+  // The current progress of a running experiment.
+  google.protobuf.DoubleValue progress = 19;
+  // The description of the experiment.
+  string description = 20;
+  // The experiment name.
+  string experiment_name = 21;
+  // The display name of the user that created the experiment.
+  string display_name = 22;
+  // The username of the user that created the experiment.
+  string username = 23;
+  // Time in seconds which experiment ran or has been running.
+  optional int32 duration = 24;
+  // The id of the project associated with this experiment.
+  int32 project_id = 25;
+  // The name of the project associated with this experiment.
+  string project_name = 26;
+  // The id of the workspace associated with this experiment.
+  int32 workspace_id = 27;
+  // The name of the workspace associated with this experiment.
+  string workspace_name = 28;
+  // The archived status of the parent project (can be inherited from
+  // workspace).
+  bool parent_archived = 29;
```

Comment:
do we need these fields?

Reply:
Same as in the earlier comment we want the experiment fields returned but not shown by default.